### ISSUE TITLE
Update react-a11y-anchors-rule to include innerimage alt text as its text content

### ISSUE THREAD

**Issue by ipip2005** [Microsoft/tslint-microsoft-contrib#319]
The text content of a link should not be only considered as the literal text in <a> element.
Below example:
```
<a href="some_href">
  <img alt="somealt" />
</a>
```
The alt attribute will be read by screen reader, it should also be treated as part on the anchor info.

As a result
1. The rule should pass when link ONLY has img tag and its alt attribute is equal or longer than 4 chars.
2. The rule should pass when links that point to same HREFs have both same literal text and alt text.
3. The rule should pass when links that point to different HREFs have either different literal text or alt text.

4. The rule should fail when link when both link's literal text and alt text is shorter than 4 chars long.
5. The rule should fail when links that point to different HREFs have same literal text and alt text.
6. The rule should fail when links that point to same HREFs have different literal text or alt text.
